### DISCUSSION TITLE
Lecture Recordings

### DISCUSSION THREAD

**Discussion by Fleischmaki** [erodola/DLAI-s2-2022#6] [General]
Hi, 
I was wondering why you chose to not record the lectures this year? I always find the recordings a fantastic tool to have another look at some of the parts I did not understand during the live session.

**Comment by erodola** [comment]
It is purely a technical reason: the zoom recordings I make have severe audio stuttering, and I couldn't find a way to fix it even with different recording tools (obs, etc.).  However, feel free to record it yourself or ask another student to record and share, no problem on my side. Also, you are free to use the recorded lectures from last year, since we are following that program closely; have a look at the DLAI-s2-2021 repository.
